perception/autoware_traffic_light_arbiter/config/traffic_light_arbiter.param.yaml

```diff
@@ -1,5 +1,6 @@
 /**:
   ros__parameters:
+    external_current_time_tolerance: 5.0
```

Comment:
How about `external_delay_tolerance` ?

Reply:
It looks good to me. I fixed in https://github.com/autowarefoundation/autoware.universe/pull/9747/commits/227d53926f6802fe4532ca53b217e2205957cf85.